cli/tests/e2e/snapshots/test_ci/test_existing_supply_chain_finding/config/base_output.txt

```diff
@@ -34,8 +34,7 @@ SEMGREP_APP_TOKEN="fake_key" SEMGREP_USER_AGENT_APPEND="pytest" SEMGREP_SETTINGS
   environment - running in environment git, triggering event is unknown
 
   CONNECTION
-  Reporting start of scan for deployment_name (scan_id=<MagicMock name='post()…
-  Fetching configuration from Semgrep Cloud Platform
+  Initializing scan (deployment=org_name, scan_id=35976)
```

Comment:
we probably don't want to hardcode a scan_id in the snapshot.

Reply:
It's hard coded in the test fixtures so this should be fine.